fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/BiomeManager.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import net.fabricmc.fabric.impl.biomes.BiomeLists;
+import net.minecraft.world.biome.Biome;
+
+public class BiomeManager
+{
+	/**
+	 * Adds the biome to spawn biomes, so that the player may spawn in the biome
+	 * 
+	 * @param biome
+	 */
+	public static void addSpawnBiome(Biome biome)
```

Comment:
Is there a need for this class, where mods could use `BiomeLists`?

Reply:
These classes add a nice user friendly wrapper around BiomeLists. I was also told that we should split the api and implementation by multiple people.